packages/core/stories/layout/flex-layout.stories.tsx

```diff
@@ -147,10 +147,7 @@ export const ContactDetailsExample = ({ index }: { index: number }) => (
       <ContactMetadataItem value="London, GBR" label="Location" />
       <ContactMetadataItem value="+44 1234 123456" label="Office" />
 
-      <ContactMetadataItem
-        value="cillum.est.exercitation@cupidatat.com"
-        label="Email"
-      />
+      <ContactMetadataItem value="cillum@cupidatat.com" label="Email" />
```

Comment:
I'm still seeing the width move? Also I don't really understand the composite usage examples?

Reply:
Should be fixed now :) The idea behind the composite usage examples is that we combine a few TK components to showcase how easily the layout components can be used, in this case we have `FlexLayout`, `Card` and `ContactDetails` although to be fair it's very similar to the Simple Usage example

Reply:
I can revisit the examples with Darrin and Ben if we think they don't make sense